src/Sidekick/Windows/LeagueOverlay/LeagueOverlayController.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using System.Windows.Input;
+using Cursor = System.Windows.Forms.Cursor;
+
+namespace Sidekick.Windows.LeagueOverlay
+{
+    public static class LeagueOverlayController
+    {
+        private static LeagueOverlayView overlayWindow;
+        private static int WindowPadding = 5;
+
+        public static bool IsDisplayed => overlayWindow.IsDisplayed;
+        public static void Show() => overlayWindow.ShowWindow();
+        public static void Hide() => overlayWindow.HideWindow();
+
+        public static void Initialize()
+        {
+            overlayWindow = new LeagueOverlayView();
+            overlayWindow.MouseDown += Window_OnHandleMouseDrag;
+        }
+
+        public static void Open()
+        {
+            if(overlayWindow == null)
+            {
+                Initialize();
+            }
+
+            var scale = 96f / Legacy.NativeProcess.ActiveWindowDpi;
+            var xScaled = (int)(Cursor.Position.X * scale);
+            var yScaled = (int)(Cursor.Position.Y * scale);
+
+            EnsureBounds(xScaled, yScaled, scale);
```

Comment:
It should appear on the main screen, at the top, and not be draggable.

Reply:
Fixed Window Show Position.
I let the window movable for now so you can move it to a second screen, etc.